pom.xml

```diff
@@ -8622,5 +8620,47 @@
                 </pluginRepository>
             </pluginRepositories>
         </profile>
+
+
+        <profile>
```

Comment:
I see no need for this profile.

Reply:
Actually it is not needed in WildFly, but it is needed in EAP side. I think it is better to have it in WildFly as well so that when new EAP branch is created from WildFly, it won't get missed.

We have internal Maven repositories used for EAP CI running within a self-controlled security environment which does not need `HTTPS`, and using `HTTPS` requires self-signed CA certificate configured in each JVM, and there is [known issue in Windows machines in brontes](https://issues.jboss.org/browse/WFCI-44)

Reply:
We don't want a way in WildFly to enable insecure downloads. This looks like a workaround to an issue so once the issue is resolved this would go away anyway.